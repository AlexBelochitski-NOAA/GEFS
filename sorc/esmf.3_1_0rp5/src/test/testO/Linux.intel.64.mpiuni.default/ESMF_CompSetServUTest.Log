20190610 112241.033931 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112241.034210 INFO     PET0  Beginning Test, file ESMF_CompSetServUTest.F90, line          68
20190610 112241.034226 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112241.035456 INFO     PET0  PASS  Creating a Component Test, ESMF_CompSetServUTest.F90, line          81
20190610 112241.035518 INFO     PET0  PASS  Setting Component Services, ESMF_CompSetServUTest.F90, line          93
20190610 112241.035534 INFO     PET0  PASS  Calling Component Init, ESMF_CompSetServUTest.F90, line         105
20190610 112241.039031 INFO     PET0  PASS  Destroying a Component Test, ESMF_CompSetServUTest.F90, line         224
20190610 112241.039057 INFO     PET0  Number of failed tests:           0
20190610 112241.039063 INFO     PET0  Ending Test, file ESMF_CompSetServUTest.F90, line         227
