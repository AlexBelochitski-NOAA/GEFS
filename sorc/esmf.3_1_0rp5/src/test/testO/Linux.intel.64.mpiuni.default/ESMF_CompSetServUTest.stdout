  Beginning Test, file ESMF_CompSetServUTest.F90, line          68
  NUMBER_OF_PROCESSORS           1
  PASS  Creating a Component Test, ESMF_CompSetServUTest.F90, line          81
  PASS  Setting Component Services, ESMF_CompSetServUTest.F90, line          93
 User initialize 1 routine called
  PASS  Calling Component Init, ESMF_CompSetServUTest.F90, line         105
  PASS  Destroying a Component Test, ESMF_CompSetServUTest.F90, line         224
  Number of failed tests:           0
  Ending Test, file ESMF_CompSetServUTest.F90, line         227
